datafusion/physical-expr/src/array_expressions.rs

```diff
@@ -2267,63 +2267,6 @@ mod tests {
         assert_eq!(result, &UInt64Array::from_value(2, 1));
     }
 
-    #[test]
```

Comment:
Remove this since `array_has_all` logic changes, and test included in sqllogic

Reply:
> Sqllogictests are more important than unit tests: unit tests are mainly for convenience.

I agree with this